cmd/ctr/commands/namespaces/namespaces.go

```diff
@@ -173,8 +173,10 @@ var removeCommand = &cli.Command{
 					}
 					log.G(ctx).WithError(err).Errorf("unable to delete %v", target)
 					continue
+				} else {
+					log.G(ctx).WithError(err).Warnf("namespace %v not found", target)
+					continue
```

Comment:
This looks like a potential change in behavior; previously we would consider the command to be idempotent; it would ignore the error if it's not found, and print the `target` (line 180/182 below).

With this change, it's no longer considered idempotent, and printing a warning; printing a warning may be ok, but the `continue` here also means that it's no longer printing the target.

I realise `ctr` is a debugging tool, so not considered "stable", but wondering if there's a potential for scripts to be depending on this (i.e. `target` to be printed).

Reply:
thanks for explaining. yes there's a potential breaking change for idempotency with this pr.